src/documentation/ToolkitDocumentation/TopicItem.tsx

```diff
@@ -248,11 +252,20 @@ const CodeEmbed = ({
   const [hovering, setHovering] = useState(false);
   // Strip the imports.
```

Comment:
This isn't really needed anymore. The code actually removing imports is clear and overall it is more nuanced now.